program/Dutch (Netherlands)/app.txt

```diff
@@ -7,20 +7,20 @@ IDS_MAIN_WNDTITLE       "Unchecky"
 IDS_HEADER_MAIN         "&Hoofdscherm"
 IDS_HEADER_CONFIG       "Instellingen"
 IDS_HEADER_ABOUT        "&Over"
-IDS_MAINCTRL_TEXT       "Welkom bij Unchecky, het programma met het doel u te beschermen tegen mogelijk ongewenste software op uw computer."
+IDS_MAINCTRL_TEXT       "Welkom bij Unchecky, het programma dat u beschermt tegen mogelijk ongewenste software op uw computer."
 IDS_MAINCTRL_TEXT_SVC_RUNNING "De Unchecky-service is actief."
 IDS_MAINCTRL_TEXT_SVC_STOPPED "De Unchecky-service is gestopt. De service zal hervatten nadat de computer is herstart."
 IDS_MAINCTRL_TEXT_QUERYERROR "Er is een fout opgetreden bij het controleren van de Unchecky-servicestatus."
-IDS_MAINCTRL_STATS_NONE_NEWGUY "Het lijkt erop dat Unchecky recentelijk is geïnstalleerd. Er zijn nog geen gegevens."
+IDS_MAINCTRL_STATS_NONE_NEWGUY "Het lijkt erop dat Unchecky onlangs is geïnstalleerd. Er zijn nog geen gegevens."
 IDS_MAINCTRL_STATS_NONE "Nog geen selectievakjes gedeselecteerd. Blijkbaar blijft u goed uit de buurt van software met ongewenste programma's!"
-IDS_MAINCTRL_STATS      "Sinds %s zijn er %u selectievakjes gedeselecteerd%s."
+IDS_MAINCTRL_STATS      "Sinds %s zijn er %u selectievakjes gedeselecteerd."
```

Comment:
This change is incorrect. The second `%s` will be replaced by the text from `IDS_MAINCTRL_STATS_WARNING`.